tutor/specs/e2e/assignment-edit.e2e.ts

```diff
@@ -0,0 +1,320 @@
+        expect(await page.$$eval('.chapter-section', (nodes: any) =>
+            nodes.map((n: any) => (n as any).innerText)
+        )).toEqual(['2.1', '2.2', '2.3'])
+
+        await page.click(':nth-match(button.ox-icon-arrow-up, 2)')
+        expect(await page.$$eval('.chapter-section', (nodes: any) =>
+            nodes.map((n: any) => (n as any).innerText)
+        )).toEqual(['2.1', '2.3', '2.2'])
+    })
+
+    it('increases selections', async () => {
+        await visitPage(page, '/course/2/assignment/edit/homework/new')
+        await disableTours()
+        await fillDetails()
+
+        await page.click('.controls .btn-primary')
+        await page.click('.chapter-checkbox .btn')
+        await page.click('.controls .btn-primary')
+        expect(await page.innerText('testEl=selection-count')).toContain('0')
+        expect(await page.innerText('testEl=selection-count-footer')).toContain('0')
+
+        await page.hover(':nth-match(.action.include, 1)', { force: true })
+        await page.waitForTimeout(100)
+        await page.click(':nth-match(.action.include, 1)', { force: true })
+        await page.waitForTimeout(10)
+
+        expect(await page.innerText('testEl=selection-count')).toContain('1')
+        expect(await page.innerText('testEl=selection-count-footer')).toContain('1')
+        expect(await page.innerText('testEl=tutor-count')).toContain('3')
+        expect(await page.innerText('testEl=tutor-count-footer')).toContain('3')
+        expect(await page.innerText('testEl=total-count')).toContain('4')
+        expect(await page.innerText('testEl=total-count-footer')).toContain('4')
+    })
+
+    it('filters question types', async () => {
+        await visitPage(page, '/course/2/assignment/edit/homework/new')
+        await disableTours()
+        await fillDetails()
+
+        await page.click('.controls .btn-primary')
+        await page.click('.chapter-checkbox .btn')
+        await page.click('.controls .btn-primary')
+        await page.click('testEl=question-type-menu')
+        await page.click('testEl=wrq-filter', { force: true })
+        expect(await page.innerText('[data-section]')).not.toContain('Written Response Questions')
+    })
+
+    it('renders external assignment', async () => {
+        await visitPage(page, '/course/2/assignment/edit/external/new')
+        await disableTours()
+        expect(await page.innerText('.heading')).not.toContain('STEP 1')
+        await page.type('[name="settings.external_url"]', 'url')
+    })
+
+    it('can add a new template', async () => {
+        const templateName = 'This is a new template'
+        await visitPage(page, '/course/2/assignment/edit/homework/new')
+        await disableTours()
+        await fillDetails()
+        await addTemplate({ name: templateName })
+        expect(await page.innerText(`testEl=${templateName}`)).toContain(templateName)
+    })
+
+    // // skipping; test is flaky in the morning on travis, some date math is incorrect here
```

Comment:
I kept this in here commented (like the cypress version) in case we can update it to work. Maybe this isn't an issue anymore with GitHub actions.